base/error.jl

```diff
@@ -197,15 +197,16 @@ windowserror(p, code::UInt32=Libc.GetLastError(); extrainfo=nothing) = throw(Mai
 """
     @assert cond [text]
 
-Throw an [`AssertionError`](@ref) if `cond` is `false`. Preferred syntax for writing assertions.
+Throw an [`AssertionError`](@ref) if `cond` is `false`. Debugging syntax for writing assertions.
```

Comment:
What is "Debugging syntax"?

Reply:
changed to "Debugging tool", which was a term used already. I think it is important to keep the "debugging" there to avoid misuse for testing assertions in production code.